src/main/java/fr/xephi/authme/security/crypts/AuthMeReloaded/src/main/java/fr/xephi/authme/security/crypts/Md5InSha1.java

```diff
@@ -0,0 +1,11 @@
+package fr.xephi.authme.security.crypts;
+
+import fr.xephi.authme.security.HashUtils;
+
+public class Md5InSha1 extends UnsaltedMethod {
```

Comment:
Could we call this class something like CMW so we know what it's for?

Reply:
It will be called CmwCrypt